libs/platform/icon-tab-bar/components/icon-tab-bar-tab/icon-tab-bar-tab.component.ts

```diff
@@ -27,13 +25,10 @@ let defaultIdIndex = 0;
 })
 export class IconTabBarTabContentDirective implements HasElementRef {
     /** Tab ID. */
-    @HostBinding('attr.id')
-    @Input()
-    id: Nullable<string>;
+    id = input<Nullable<string>>();
```

Comment:
you need to handle the host binding for the `id`, because now it's no longer rendered.